src/View/ViewContainer.vala

```diff
@@ -334,17 +334,17 @@ namespace Marlin.View {
 
             switch ((Marlin.OpenFlag)flag) {
                 case Marlin.OpenFlag.NEW_TAB:
-                    window.open_single_tab (loc, view_mode);
```

Comment:
Is a case environment doing (apparently) nothing desired? Or am I thinking something wrong?